### DISCUSSION TITLE
Is it possible to create a (Multilevel) Layer Filter?

### DISCUSSION THREAD

**Discussion by tibohu** [mozman/ezdxf#861] [Q&A]
Hi, Layer Filters can be very useful. Sometimes essential. I did not found too much detail about it in the docs. Could someone share working code examples?

**Comment by mozman** [comment]
No, this CAD feature is not supported.

**Reply by tibohu** [reply]
Hi Mozman, thank you for your answer. I am an occasional/beginner programmer so would definitely need your guidance, but would be happy to contribute to the Layer Filters. Please, let me know if you are interested!
